webpack/assets/javascripts/bundle.js

```diff
@@ -4,7 +4,8 @@ require('expose?_!lodash');
 require('expose?jstz!jstz');
 require('jquery.cookie');
 require('./bundle_flot');
+require('script!../../../node_modules/patternfly/dist/js/patternfly.js');
```

Comment:
note: afair latest patternfly changed their directory structure, maybe we need to specify a minimum version? (see https://github.com/patternfly/rcue/pull/48)